subjects/loafofbread/README.md

```diff
@@ -0,0 +1,44 @@
+# loafofbread
+
+### Instructions
+
+Write a function `LoafOfBread()` that takes a string and returns another one with words of 5 characters and skips the next character followed by newline `\n`.
+
+- If there is a space in the middle of a word it should ignore it and get the first character until getting to a length of 5.
```

Comment:
I think it would be more accurate if you rephrase to `...and get the next character until getting...`, what do you think?

Reply:
I think it is more accurate as you suggested.
It's fixed.